report_qweb_element_page_visibility/views/layouts.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+<data>
+
+  <template id="minimal_layout" inherit_id="report.minimal_layout">
+    <xpath expr="//t[@t-if='subst is True']" position="replace">
+    	<t t-if="subst is True">
+        <script>
+            function subst() {
+                var vars = {};
+                var x = document.location.search.substring(1).split('&amp;');
+                for (var i in x) {
+                    var z = x[i].split('=', 2);
+                    vars[z[0]] = unescape(z[1]);
+                }
+                var x=['frompage', 'topage', 'page', 'webpage', 'section', 'subsection', 'subsubsection'];
+                for (var i in x) {
+                    var y = document.getElementsByClassName(x[i]);
+                    for (var j=0; j&lt;y.length; ++j)
+                        y[j].textContent = vars[x[i]];
+                    }
+                var operations = {
+                    'not-first-page': function (elt) {
+                        elt.style.visibility = (vars.page === vars.frompage) ? "hidden" : "visible";
+                    },
+                    'not-last-page': function (elt) {
+                        elt.style.visibility = (vars.page === vars.topage) ? "hidden" : "visible";
+                    },
+                    'first-page': function (elt) {
+                        elt.style.visibility = (vars.page === vars.frompage) ? "visible" : "hidden";
+                    },
+                    'last-page': function (elt) {
+                        elt.style.visibility = (vars.page === vars.topage) ? "visible" : "hidden";
+                    },
+                };
+                for (var klass in operations) {
+                    var y = document.getElementsByClassName(klass);
+                    for (var j=0; j&lt;y.length; ++j)
+                        operations[klass](y[j]);
+                    }
```

Comment:
Totally stylistic/linty, but can you please move the `}` to be inline with its associated `for` statement? I had to stare at this way too long to realize the `for` it's matched with doesn't have a `{` and that there are in fact no missing brackets